RobotCode/snobot2018/simulator_config/simulator_config_ctre.yml

```diff
@@ -39,20 +39,10 @@ mPwm:
     mMotorType: CIM
```

Comment:
Another issue was the simulator config.  Made the motors a more realistic speed and removed unnecessary stuff